.github/workflows/docs.yaml

```diff
@@ -1,6 +1,10 @@
 name: Documentation Updates
 
-on: [push]
+on:
+  push:
+    paths: ["Documentation/**"]
+  pull_request:
+    paths: ["Documentation/**"]
```

Comment:
@errordeveloper Do you know if the following works?
```yaml
- [push,pull_request]:
    paths: ["Documentation/**"]
```

Reply:
I'm pretty sure it won't. The `[push,pull_request]` would be to weird as a key name. I was actually hoping that the following may work:
```
  - push:
      paths: ["Documentation/**"]
  - pull_request:
      paths: ["Documentation/**"]
```
But it didn't. Personally, I find ambiguity of config files like this and magic shortcut very annoying, how much time people have to spend guessing! The only sensible answer would be to ditch JSON/YAML and use a scripting language, but then GitHub would have to support at least a few different languages...